corehq/apps/accounting/models.py

```diff
@@ -1964,13 +1965,24 @@ class Meta(object):
 
     _pdf = None
 
+    @property
+    def emailed_to(self):
+        return ','.join(self.emailed_to_list)
+
     @property
     def recipients(self):
-        return self.emailed_to.split(',') if self.emailed_to else []
+        return self.emailed_to_list
 
     @recipients.setter
     def recipients(self, emails):
-        self.emailed_to = ','.join(emails)
+        self.emailed_to_list = emails
+
+    def add_recipients(self, emails):
+        if self.emailed_to_list and self.emailed_to_list != []:
+            for email in emails:
+                self.emailed_to_list.append(email)
+        else:
+            self.emailed_to_list = emails
```

Comment:
I think `emailed_to_list` will always be a list already, so you could probably just write:
```python
def add_recipients(self, emails):
    self.emailed_to_list.extend(emails)
```

Reply:
Also, could duplicates be an issue here?

Reply:
I don't think duplicates are a concern.